.eslintrc

```diff
@@ -1,3 +1,9 @@
 {
-  "extends": "eslint-config-mitodl"
+  "extends": "eslint-config-mitodl",
+  "settings": {
+    "react": {
+      "version": "15.6.1",
+      "flowVersion": "0.59.0" // Flow version
```

Comment:
Is it possible to put these settings in the eslint-config-mitodl repo somehow?

Reply:
I don't think we'd want to, because then we'd have to be on the same version of react and flow everywhere (and sometimes we run into difficulties upgrading them)